tools/ui-components/src/alert/index.ts

```diff
@@ -1 +1,2 @@
 export { Alert } from './alert';
+export type { AlertProps } from './alert';
```

Comment:
For future reference, we have an infrequent job that clears all unused exports. So, unless something is "consumed", it should not be "just made to be exported".

I am not sure if this is already imported elsewhere.

Reply:
I can see the ui-component become unusable because of that, will this apply to ui-component